src/components/Button/Button.styled.ts

```diff
@@ -471,6 +471,24 @@ export const types = {
 	    line-height: ${textButtonStyles.lineHeight};
     }
   `,
+	black_text_link: styled(BlackOnWhite)`

```

Comment:
As I mentioned in the comment from Button.tsx file, this whole new style is not necessary since the component already makes the transformations enough to what you need. I made some small changes and this is what I ended up needing to style it the same way: 
![image](https://user-images.githubusercontent.com/23409284/176962191-d44a7ffb-967a-4321-8a4d-9b9c061ccbc8.png)

![image](https://user-images.githubusercontent.com/23409284/176962248-d28cc62b-84ca-48ec-a3c6-3f74c51da5fa.png)
This is for styling the icon, is at the beginning of the file.]

Also, it seems that the original style you have is missing a space between the icon and the text but with these new changes I believe you won't need to add them.